docs/aerodromes/classc/Gingin.md

```diff
@@ -4,7 +4,36 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login ID              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| **Gingin ADC**    | **Gingin Tower**  | **118.500**    | **GIG_TWR**        |
+
+## Airsapce
+ ADC owns the Class C airspace **in the PE CTR** within 5nm of the YPEA ARP from `SFC` to `A035`. This airspace is primarily used for military circuits and initial and pitch approaches.
+
+## Miscellaneous
+
+### Circuit Operations 
+The Circuit Area Airspace is allocated to be within 5nm of the YGIG ARP from `SFC` to `A035` exculding the area within 12TAC Pearce. Aircraft can be instructed to extend outside of this airspace by ATC for traffic management. Hawks have circuit altitude of A016 while both 2FTS and RSAF PC21s are assigned A012.
+
+### Circuit Direction
+The Ciruits are preferred to be to the south of the field.
+
+### Initial and Pitch Procedures 
+The intial points for Gingin are RWY 08 is the corner of the pine plantation and RWY 26 is the paddock divided by the creek. The intial point is 1500ft for both Hawks and PC21's.
```

Comment:
This might be a good candidate for a table. Whilst IRL they get Hawks and PC21s, online it's likely that there'll be all sorts of aircraft, so I'm happy if we want to just nominate a single initial altitude

Reply:
Good candidate for a diagram

Reply:
> This might be a good candidate for a table. Whilst IRL they get Hawks and PC21s, online it's likely that there'll be all sorts of aircraft, so I'm happy if we want to just nominate a single initial altitude

Will do 1500ft